City-of-Helsinki/berth-reservations-admin
# VEN-673 | Implement BoatEditForm

## Description :sparkles:

Implement form for editing boat information. Additionally refactors common form scss and updates snapshots to use renderer.

### Closes :no_good_woman:

[VEN-673](https://helsinkisolutionoffice.atlassian.net/browse/VEN-673)

## Testing :alembic:

### Automated tests :gear:️

- `src/features/customerView/forms/boatForm/__tests__/BoatForm.test.tsx`
- Updated relevant snapshots

### Manual testing :construction_worker_man:

1. Open the customers page
2. Select a customer that has a boat
3. The boat's information should be editable
4. mutations should be sent to the BE
